docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -217,83 +370,53 @@
   },
   {
    "cell_type": "markdown",
-   "id": "e963d17c-e6e1-461d-a89d-7f4256643c9d",
    "metadata": {},
    "source": [
-    "#### QURI Chemistry with UCCSD\n",
-    "\n",
-    "UCCSD is a common chemistry-inspired ansatz. In the QURI Chemistry function, you can customize it with the `ansatz_setting` key. Options for customizing UCCSD is summarized in the table.\n",
-    "\n",
-    "| Name             | Type | Description                                                                                | Required | Default | Example |\n",
-    "| :--------------- | :--- | :----------------------------------------------------------------------------------------- | :------- | :------ | :------ |\n",
-    "| use_singles      | bool | If True, all the single excitation modes are included. Otherwise, they are all turned off | No       | True    | True    |\n",
-    "| n_trotter        | int  | The number of Trotter steps for the exponentiated excitation operator                     | No       | 1       | 1       |\n",
-    "| reduce_parameter | bool | Only True is allowed for QSCI. refer to the `singlet_excitation` field in the [QURI Parts documentation](https://quri-parts.qunasys.com/api/quri_parts/openfermion/quri_parts.openfermion.ansatz#quri_parts.openfermion.ansatz.TrotterUCCSD)                                                            | No       | True    | True    |\n",
-    "\n",
-    "A dictionary for running QURI Chemistry with UCCSD:"
+    "Next, set up the ansatz. We pick the `DoubleExcitation` as the ansatz, as it carries the most non-trivial information with the lowest circuit depth. You can specify how many excitation amplitudes to include with the `n_amplitudes` key in the `ansatz_setting` field. The default value is 10. The function sorts the CCSD amplitudes according to their magnitude and chooses the `n_amplitudes` largest for constructing the ansatz. An example JSON dictionary using this ansatz follows:"
```

Comment:
```suggestion
    "Next, set up the ansatz. This example uses the `DoubleExcitation` as the ansatz because it carries the most non-trivial information with the lowest circuit depth. You can specify how many excitation amplitudes to include with the `n_amplitudes` key in the `ansatz_setting` field. The default value is 10. The function sorts the CCSD amplitudes according to their magnitude and chooses the `n_amplitudes` largest for constructing the ansatz. An example JSON dictionary using this ansatz follows:"
```